sci_bert/models/ner_model.py

```diff
@@ -0,0 +1,252 @@
+from typing import Dict, Optional, List, Any
```

Comment:
same as above, do we need to copy the model code here or can we work with the one defined in allennlp while customizing it using the configuration file ? It is possible that the configuration file customizations are not enough, and in that case it makes sense to copy it.